calc/calc.py

```diff
@@ -52,6 +58,72 @@ def __init__(self, text, position=0, current_token=None):
         self.position = position
         self.current_token = current_token
 
+    def _tokenize_integer(self, text):
+        """If have found a single digit then this helper function will read
+        characters until the end of the integer is found.
+
+        args:
+            text: A section of the text that is assumed to contain an integer
+
+        Returns:
+            Token
+
+        Raises:
```

Comment:
Odd, I added it because there were methods that had that in the doc strings.

[Like this one](https://github.com/rawrgulmuffins/red-green-refactor/blob/f28b5b945e8c192644a65703e902d4870c2ced92/calc/calc.py#L138)